### ISSUE TITLE
[D2 summon] Loot (chest) bugged randomly

### ISSUE THREAD

**Issue by phonkala** [chromiecraft/chromiecraft#3327]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [X] 50-59
- [X] 60

### Current Behaviour

Have seen few bugs with the loot chest after summoning Sothos and Jarien in Stratholme after last boss, using Brazier of Invocation. It might be more generic issue though with loot.

One time the chest never appeared after killing the summoned bosses.

Another time we rolled for an item (hunting the +hit offhand myself) but the roll never ended, chest disappeared and neither of us that need rolled got the item. 3 people did leave earlier but they greed rolled, us that need rolled stayed 5-10min and neither of us ever got the loot.

I have seen the roll problem with other loot too, sometimes items just don't get rolled so that might be related to other issues. Seems like sometimes the roll just never is finished in general.

### Expected Blizzlike Behaviour

Well it's probably easy to guess how it is supposed to go, loot rolls should end someone winning the roll and of course if loot chest doesn't appear after killing the bosses, it's bugged too.

### Steps to reproduce the problem

This seems to happen very randomly, I have been farming those mentioned bosses for maybe 20-30 runs and only once the chest didn't even appear. Loot rolls staying unfinished I've seen few times (on other dungeons and while playing other character too) so hard to define how to reproduce this consistently and what is the actual bug here.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/48fccc851d8491c9019a585c74417641376795de

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Gultask** [comment]
I wasn't able to reproduce, but since it's random as you say it would indeed be hard to reproduce.
Though, apparently, there were some issues with chest group rolls and this fight in particular, see: https://github.com/azerothcore/azerothcore-wotlk/issues/9865 and https://github.com/chromiecraft/chromiecraft/issues/2650

**Comment by temperrr** [comment]
I have sadly also not found a way to reproduce this issue. If you have more information to properly reproduce, feel free to make a new issue.
